src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/ContextBuilder/workingListsContextBuilder.types.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import type {
+    AddTemplate,
+    CancelLoadView,
+    Categories,
+    ColumnConfigs,
+    DeleteTemplate,
+    DataSource,
+    LoadedContext,
+    LoadView,
+    SelectTemplate,
+    UnloadingContext,
+    UpdateList,
+    UpdateTemplate,
+    WorkingListTemplates,
+    WorkingListTemplate,
+    WorkingListsOutputProps,
+} from '../workingLists.types';
+import type {
+    ChangePage,
+    ChangeRowsPerPage,
+    ClearFilter,
+    CustomRowMenuContents,
+    FiltersData,
+    SelectRestMenuItem,
+    SelectRow,
+    SetColumnOrder,
+    Sort,
+    StickyFilters,
+    UpdateFilter,
+} from '../../../../ListView';
+
+type ExtractedProps = $ReadOnly<{|
```

Comment:
Thinking if we could reuse the `EventWorkingListsReduxOutputProps` as you did somewhere else with the something you called base?

Reply:
Keeping this one as it is because this is the pattern I have been using throughout.